src/Mvc/dotnet-getdocument/src/Commands/InvokeCommand.cs

```diff
@@ -16,155 +16,123 @@ internal class InvokeCommand : HelpCommandBase
                             {
-                                var projectAssets = JToken.ReadFrom(reader);
-                                var packageFolders = projectAssets["packageFolders"]
-                                    .Children<JProperty>()
-                                    .Select(p => p.Name);
-
-                                foreach (var packageFolder in packageFolders)
-                                {
-                                    args.Add("--additionalProbingPath");
-                                    args.Add(packageFolder.TrimEnd(Path.DirectorySeparatorChar));
-                                }
+                                args.Add("--additionalProbingPath");
+                                args.Add(packageFolder.TrimEnd(Path.DirectorySeparatorChar));
                             }
                         }
 
-                        if (File.Exists(project.ProjectRuntimeConfigFilePath))
+                        var runtimeConfigPath = Path.ChangeExtension(assemblyPath, ".runtimeconfig.json");
+                        if (File.Exists(runtimeConfigPath))
                         {
                             args.Add("--runtimeConfig");
-                            args.Add(project.ProjectRuntimeConfigFilePath);
+                            args.Add(runtimeConfigPath);
                         }
-                        else if (!string.IsNullOrEmpty(project.RuntimeFrameworkVersion))
+                        else
                         {
-                            args.Add("--fx-version");
-                            args.Add(project.RuntimeFrameworkVersion);
+                            var runtimeFrameworkVersion = _projectOptions.RuntimeFrameworkVersion.Value();
+                            if (!string.IsNullOrEmpty(runtimeFrameworkVersion))
+                            {
+                                args.Add("--fx-version");
+                                args.Add(runtimeFrameworkVersion);
+                            }
                         }
 
                         args.Add(Path.Combine(toolsDirectory, InsideManName + ".dll"));
                         break;
 
                     case ".NETStandard":
-                        throw new CommandException(Resources.FormatNETStandardProject(project.ProjectName));
+                        throw new CommandException(Resources.FormatNETStandardProject(projectName));
 
                     default:
                         throw new CommandException(
-                            Resources.FormatUnsupportedFramework(project.ProjectName, targetFramework.Identifier));
+                            Resources.FormatUnsupportedFramework(projectName, targetFramework.Identifier));
                 }
 
                 args.AddRange(_args);
                 args.Add("--assembly");
-                args.Add(project.TargetPath);
+                args.Add(assemblyPath);
+                args.Add("--project");
+                args.Add(projectName);
```

Comment:
I'm probably not the best person to review the inside-man stuff. I've never really known how this works 😕